ntbs-service/Services/NotificationChangesService.cs

```diff
@@ -196,6 +196,19 @@ private static IEnumerable<AuditLog> GetCanonicalLogs(List<AuditLog> group)
                 yield break;
             }
 
+            // Adding treatment started information creates both clinical details treatment event updates. We want to
+            // show both, as the change is done on the clinical details page but seen on the treatment events section
+            if(group.Count == 2
+               && group.Any(log => log.EntityType == nameof(TreatmentEvent))
+               && group.Any(log => log.EntityType == nameof(ClinicalDetails)))
+            {
+                foreach (var log in group)
+                {
+                    yield return log;
+                }
+                yield break;
+            }
```

Comment:
Can we add a test for this?

Reply:
I have done, I also added verification of the logger not being called for the other tests in the suite